documentation/docs/14-configuration.md

```diff
@@ -199,7 +199,8 @@ An object containing zero or more of the following values:
 
 ### ssr
 
-Enables or disables [server-side rendering](#ssr-and-javascript-ssr) app-wide.
+- `true` or `false` — Enables or disables [server-side rendering](#ssr-and-javascript-ssr) app-wide.
+- `"never"` — Prevents all pages from being evaluated on the server, on both [server-side rendering](#ssr-and-javascript-ssr) and [prerendering](#ssr-and-javascript-prerender). Using this option is recommended when building an SPA.
```

Comment:
Might be worth explaining the difference between `false` and `"never"` here: `ssr: false` in the config still allows individual pages to select `export const ssr = true`, but `ssr: "never"` in the config will completely ignore `export const ssr = whatever` in pages. Perhaps in the "`true` or `false`" bullet point you could add the sentence "Page-level `ssr` exports can override this value", and in the `"never"` bullet point you can explain that page-level `ssr` configs are ignored if `"never"` is chosen at the config level.